modules/lvms-scaling-storage-of-clusters-using-cli.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-cli_{context}"]
+= Scaling up the storage of clusters by using the CLI
+
+You can scale up the storage capacity of the worker nodes on a cluster by using the `OpenShift` CLI.
+
+.Prerequisites
+
+* You have additional unused devices on each cluster to be used by {lvms}.
+* You have installed the OpenShift CLI (`oc`).
+* You have created an `LVMCluster` CR.
+
+.Procedure
+
+. Edit the `LVMCluster` custom resource (CR) YAML file by running the following command:
+[source, terminal]
+----
+$ oc edit <lvmcluster_file_name> -n <namespace>
+----
+
+. Add the path to the new device in the `deviceSelector` field of the `LVMCluster` CR YAML file.
++
+include::snippets/lvms-scaling-up-storage-lvmcluster-cr-snippet.adoc[]
+
+. Save the `LVMCluster` CR YAML file
```

Comment:
```suggestion
. Save the `LVMCluster` CR YAML
```
This will already apply the changes. We don't need the `oc apply` command here. It is necessary when you make the changes on a local file and then you want to apply the changes to the cluster.

Reply:
Updated as per the suggestion.